src/_store/AppInfoProxy.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import LoadingView from '../loading-view/LoadingView';
+
+@connect(state => ({ appInfo: state.appInfo }))
+export default class AppInfoProxy extends Component {
+    static propTypes = {
```

Comment:
Not sure if this is a poor name, suggestion are welcome
We can later tap into this component to provide additional information if we necessary ~

I kinda over-engineered it as I was thinking to show how many time they failed, and possibly ask them to check their connection if failure exceeded some threshold, but then found out ws failure was not handled well from the library, so I'll leave it here at the moment